signup/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓 회원가입</title>
+  <link rel="icon" href="/images/logo/favicon.ico">
+  <link rel="stylesheet" href="/styles/global.css">
+  <link rel="stylesheet" href="/styles/login.css">
+</head>
+<body>
+  <header>
+    <a href="/">
+      <img class="logo-big" alt="판다마켓 홈" src="/images/logo/panda-market-logo.png">
+    </a>
+  </header>
+  <main>
+    <form id="signup-form">
+      <label class="label" for="email">이메일</label>
+      <input class="input" type="email" id="email" placeholder="이메일을 입력해주세요">
+
+      <label class="label" for="nickname">닉네임</label>
+      <input class="input" type="text" id="nickname" placeholder="닉네임을 입력해주세요">
+
+      <label class="label" for="password">비밀번호</label>
+      <input class="input" type="password" id="password" placeholder="비밀번호를 입력해주세요">
+
+      <label class="label" for="password-check">비밀번호 확인</label>
+      <input class="input" type="password" id="password-check" placeholder="비밀번호를 다시 한 번 입력해주세요">
+
+      <button class="button button-large">회원가입</button>
+
+      <div class="quick-login">
+        <p>간편 로그인하기</p>
+        <div class="quick-login-sns">
+          <a href="https://www.google.com/">
+            <img class="quick-login-sns-icon" src="/images/social/google-icon.png">
+          </a>
+          <a href="https://www.kakaocorp.com/page/">
+            <img class="quick-login-sns-icon" src="/images/social/kakaotalk-icon.png">
+          </a>
+        </div>
+      </div>
+
+      <div class="login"> 
+        <span>이미 회원이신가요?</span>
```

Comment:
참고로 p 요소와 span 요소의 차이는 무엇일까요?
참고링크: https://inpa.tistory.com/entry/HTML-%F0%9F%93%9A-p-div-span-%ED%83%9C%EA%B7%B8-%EC%A0%95%EB%A6%AC#%E2%80%8B%3Cp%3E

Reply:
p가 문단이고 span이 짧은 조각이라고 생각해서 이렇게 작성했는데,
- div: 영역 구분 목적
- p: 글자 표현 목적
- span: 글자들 중 일부에 특별한 스타일 적용할 목적
이렇게 구분하면 될까요?
560874 refactor(text tag): Use <p> instead of <span> to express text